lib/todo/executor.rb

```diff
@@ -0,0 +1,9 @@
+require 'todo'
+
+module Todo
+  Executor = CLI::Kit::Executor.new(
```

Comment:
This thing we did with Config to work around new and old ways feels like an anti-pattern that I'd rather not perpetuate.

Reply:
I feel like this could be a default or something, I'm having a hard time following the api split here

Reply:
How should we accomplish this?

* We have a module of functionality, the `Executor`.
* It needs three bits of information provided:
    * Name of tool
    * Command Registry
    * Log file

Options I can think of:

* Subclass in app, require redefinition of these three methods.
* Statically assign these values in the `CLI::Kit` singleton instance
* Require application to pass configuration values to each API call on the Executor, rather than on on initialization

---

We talked about this a bunch in person and have come to a rough consensus that this is a probably-reasonable goal, but the implementation still feels a little off because, while `Executor` isn't truly a class or module, it is masquerading as one (although it is fair to treat it as a module because of the way it is). We discussed assigning it in screaming snake case, but that makes one wonder whether it's a module or a primitive, which is another tradeoff. Assigning it as a class method on `Todo` is also not a lie, but `Todo.executor.call` doesn't convey the static-ness of `Todo::Executor`. I'll try to find a better way.